packages/devextreme/testing/testcafe/model/dataGrid/groupPanelItem.ts

```diff
@@ -0,0 +1,13 @@
+const FILTER_BUTTON_SELECTOR = '.dx-header-filter';
```

Comment:
Let's change this const to the ```CLASS``` const
To make it uniform with other our test models: 
[link to the example](https://github.com/DevExpress/DevExtreme/blob/24_1/packages/devextreme/testing/testcafe/model/dataGrid/index.ts#L24)